src/homework_5/robotConnection/RobotApp.java

```diff
@@ -0,0 +1,31 @@
+package homework_5.robotConnection;
+
+public class RobotApp {
+    static RobotConnection robotConnection;
+
+    public static void main(String[] args) {
+        try {
+            moveRobot(new RobotConnectionManagerImpl(), 5, 8);
+        } catch (RobotConnectionException e) {
+            e.getMessage();
+        }
+    }
+
+    public static void moveRobot(RobotConnectionManager robotConnectionManager, int toX, int toY) {
```

Comment:
is this method work on stepik?
To make this method work  there, you need to add try-catch in the finally block. And you can remove:
catch (Exception e) {
                break;
            }